tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java

```diff
@@ -418,8 +426,12 @@ else if( elementName.equals( ELEMENT_LANGUAGE_PROPERTIES ) )
     public static List<IFile> getDefaultLanguagePropertiesFromProject( IProject project )
     {
         final ILiferayProject lp = LiferayCore.create( project );
-
-        if( lp != null )
+        
+        if( lp!= null && lp.getClass().getSimpleName().equals( "LiferayGradleProject" ))
```

Comment:
We need to find a more robust way to detect this, not hardcoding the Classname in this core plugin class.